scripts/qa_crowbarsetup.sh

```diff
@@ -3484,10 +3484,10 @@ function oncontroller_testsetup()
     fi
 
     # cleanup so that we can run testvm without leaking volumes, IPs etc
-    nova remove-floating-ip "$instanceid" "$floatingip"
-    nova floating-ip-delete "$floatingip"
-    nova stop "$instanceid"
-    wait_for 100 1 "test \"x\$(nova show \"$instanceid\" | perl -ne 'm/ status [ |]*([a-zA-Z]+)/ && print \$1')\" == xSHUTOFF" "testvm to stop"
+    #nova remove-floating-ip "$instanceid" "$floatingip"
+    #nova floating-ip-delete "$floatingip"
+    #nova stop "$instanceid"
```

Comment:
If you comment out the `stop`, then I think you can drop the `start` we do on reboot. Or is it still needed?